include/plat/marvell/a8k/common/board_marvell_def.h

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
nit: When the copyright issue is sorted, just have one header in this file (and the others that have 2 headers).

Reply:
Any news regarding the copyright? If it is a problem to change it, we could accept it as it is. We mostly do this for consistency across all codebase (if not we have to start adding "parts copyright Arm.."). Maybe you could remove the link to the SPDX website, though, but that's just a nit if you think it's worth your time.

Reply:
Unfortunately the legal department demanded me to fill a formal request, which will take me some time to create and submit. A simple email query did not work for this case. At this moment I do not expect to close this issue shortly, but doing my best to complete all mandatory steps on  my side.
As for link to SPDX site - in the past it was requested by our legal in exchange to the full BSD 3-clause license text removal from the copyright header. Legal people have different view to all this stuff then us, technical crowd :-)

Reply:
Thanks Antonio for the efforts that you put in reviewing this patchset. 
It will be great if you could accept keeping the original Marvell copyright as you suggested.

Reply:
Yeah, I agree that legal stuff can be tricky to get right. We can accept the changes as they are now.